<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="LoginTab">
       <attribute name="title">
        <string>Login</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contrase√±a:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LE_Username">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>180</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>190</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LE_Password">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>270</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Login">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>510</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Iniciar Sesion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>30</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>350</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ingresar como:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_tipo_usuario">
        <property name="geometry">
         <rect>
          <x>324</x>
          <y>423</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Docente</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coordinador</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IEDD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Consultor</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="DocenteTab">
       <attribute name="title">
        <string>Docente</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Docente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Logout">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>480</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cerrar Sesion</string>
        </property>
       </widget>
       <widget class="QLabel" name="imageLabel">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>140</y>
          <width>301</width>
          <height>261</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Coordinador</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Coordinador</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Logout_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>480</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cerrar Sesion</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>IEDD</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>IEDD</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Logout_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>460</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cerrar Sesion</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Consultor</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>40</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Consultor</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Btn_Logout_4">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>470</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cerrar Sesion</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
